test/test_runner_battle.c

```diff
@@ -1536,7 +1536,7 @@ static u32 GenerateNature(u32 nature, u32 offset)
 
 void ClosePokemon(u32 sourceLine)
 {
-    s32 i;
+    int i;
```

Comment:
do we need to use `int` over `s32` here?

Reply:
Nope, I think I changed it by accident.